src/new-client/package.json

```diff
@@ -95,6 +96,8 @@
     "@types/styled-components": "^5.1.9",
     "@typescript-eslint/eslint-plugin": "^4.20.0",
     "@typescript-eslint/parser": "^4.20.0",
+    "@types/react-window": "1.8.5",
```

Comment:
I don't see these packages as a dependency. Were they removed, or are these needed as a peer dependency of virtuoso?

Reply:
Forgot to delete the types, they were for the previous iterations while I was trying to get those libraries to work here. Fixed. As for the comments above, fixed warnings and adjusted styles